src/Nethermind/Directory.Build.props

```diff
@@ -9,7 +9,7 @@
     <Copyright>Demerzel Solutions Limited</Copyright>
     <Product>Nethermind</Product>
     <SourceRevisionId Condition="'$(Commit)' != ''">$(Commit.Substring(0, 8))</SourceRevisionId>
-    <VersionPrefix>1.20.0-unstable</VersionPrefix>
+    <VersionPrefix>1.21.0-unstable</VersionPrefix>
```

Comment:
Should be:

```csproj
<VersionPrefix>1.21.0</VersionPrefix>
<VersionSuffix>unstable</VersionSuffix>
```

Reply:
@rubo Is this some recent change? we have this approach for some time already.

Reply:
Where commit will be? Still it be like 1.21.0-unstable+commit?

Reply:
No changes in version format. This is just the way to do it. I've always missed this PRs to review. This time I caught it ;)